### PR TITLE
Fixes all ports to be `uint16`

### PR DESCRIPTION
**Pull request by oxisto**
Some code was incorrectly using `int16` for ports (which truncates) it or `int` (which is ok but unncessary). This PR addresses most of it. 

This also includes a small ontology update from https://github.com/clouditor/cloud-property-graph/pull/45

### CODE REVIEW

**Review by anatheka — APPROVED** [review]

**Review by lebogg — COMMENTED** [review]
Changes done LGTM, but I think you missed some:
- engine.go: : Default Ports in the constants section
- cloud_test.go (grpcPort)
- cli.go (grpcPort)
- auth.go (StartAuthenticationServer)

**Review by lebogg — CHANGES_REQUESTED** [review]
see above

**Comment by oxisto** [comment]
> Changes done LGTM, but I think you missed some:
> 
> * engine.go: : Default Ports in the constants section
> * cloud_test.go (grpcPort)
> * cli.go (grpcPort)
> * auth.go (StartAuthenticationServer)

I have converted the rest of the ports to `uint16` and also used the new `netip.AddrPort` which also uses `uint16` for ports for more consistentcy.

File: internal/testutil/auth.go [hunk 1]
```diff
@@ -45,24 +45,24 @@ func StartAuthenticationServer() (srv *oauth2.AuthorizationServer, port int, err
 		_ = srv.Serve(nl)
 	}()
 
-	port = nl.Addr().(*net.TCPAddr).Port
+	port = uint16(nl.Addr().(*net.TCPAddr).Port)
```
Comment (lebogg) [inline internal/testutil/auth.go:48]: Maybe using `netip.AddrPort` here as well instead of cast?

**Review by lebogg — COMMENTED** [review]
One change missing

**Review by lebogg — APPROVED** [review]

**Pull request merged**